.github/workflows/integration.yml

```diff
@@ -0,0 +1,83 @@
+name: Integration Test
+
+on:
+    pull_request:
+        branches: [ master ]
+
+jobs:
+    fedora_basic:
+        runs-on: ubuntu-latest
+        timeout-minutes: 45
+        concurrency:
+            group: ${{ github.workflow }}-${{ github.ref }}-${{ matrix.container }}-${{ matrix.test }}
+            cancel-in-progress: true
+        strategy:
+            matrix:
+                container: [
+                        "fedora:33",
+                        "fedora:34",
+                ]
+                test: [
+                        "04",
+                        "21",
+                        "36",
+                ]
+            fail-fast: false
+        container:
+            image: ghcr.io/dracutdevs/${{ matrix.container }}
+            options: "--privileged -v /dev:/dev"
+        steps:
+            -   name: "Checkout Repository"
+                uses: actions/checkout@v2
+                with:
+                    fetch-depth: 0
+
+            -   name: "${{ matrix.container }} TEST-${{ matrix.test }}"
+                run: ./tools/test-github.sh "TEST-${{ matrix.test }}" ${{ matrix.test }}
+    test:
+        runs-on: ubuntu-latest
+        timeout-minutes: 45
+        concurrency:
+            group: ${{ github.workflow }}-${{ github.ref }}-${{ matrix.container }}-${{ matrix.test }}
+            cancel-in-progress: true
+        strategy:
+            matrix:
+                container: [
+                        "fedora:latest",
+                        "opensuse:latest",
```

Comment:
`opensuse:latest` has been deprecated for quite a while now noted in the description. I am not even sure what state this is. We should be using `opensuse/tumbleweed:latest`  and probably add `opensuse/leap:latest`, too.

Reply:
it is in fact opensuse/tumbleweed:latest

Reply:
https://github.com/dracutdevs/dracut/blob/5a04bd2dd3a091af333c534f4534c87c5175fe13/test/container/Dockerfile-OpenSuse-latest#L1

Reply:
my bad, thanks!